src/NServiceBus.SagaAudit.Tests/SagaEntitySerialization.cs

```diff
@@ -47,13 +48,13 @@ public void Saga_entity_compatible_with_serviceinsight()
                     IntProperty = 1
                 }
             };
-            var sagaDataJson = SimpleJson.SerializeObject(entity, new SagaEntitySerializationStrategy());
+            var sagaDataJson = JsonSerializer.Serialize(entity);
 
             var sagaDataProperties = JsonPropertiesHelper.ProcessValues(sagaDataJson);
 
-            var jsonObj = SimpleJson.DeserializeObject(sagaDataJson) as JsonObject;
+            var jsonObj = JsonSerializer.Deserialize<JsonObject>(sagaDataJson);
 
-            Assert.IsNotNull(jsonObj, "SimpleJson.DeserializeObject");
+            Assert.IsNotNull(jsonObj);
 
             foreach (var p in sagaDataProperties)
```

Comment:
@SzymonPobiega @boblangley I don't fully get this test and your names showed up in git blame :)

This PR changes from simplejson to system json, does these changes look ok to you?